.github/workflows/tsan.yaml

```diff
@@ -266,7 +258,6 @@ jobs:
           export PYTHON_SHA256=($(sha256sum ${GITHUB_WORKSPACE}/python-tsan.tgz))
           echo "Python sha256: ${PYTHON_SHA256}"
 
-          python3 -VV
```

Comment:
This line was helpful to get exact version of python under the CI run

Reply:
Yeah, but we don't really care that much! We don't use it for anything but the `build.py` script, and that should run on any Python 3.